Makefile

```diff
@@ -13,7 +13,13 @@
 # limitations under the License.
 
 COMPILER?=g++
+
+GCC_VER_GTE53 := $(shell echo `gcc -dumpversion | cut -f1-2 -d.` \>= 5.3 | sed -e 's/\./*100+/g' | bc )
```

Comment:
@dascandy Thanks! You save for me a lot of time and nerves. But I changed solution about how to make alias for boost/filesystem or std. I think new solution is more understandable.